### ISSUE TITLE
Sanity check config

### ISSUE THREAD

**Issue by Fear-in-a-hand-full-of-dust** [67372a/LoRA_Easy_Training_Scripts#23]
Could we get some sort of sanity check config? One of the latest updates broke my existing config and it'd be nice to have a "works for sure" baseline.

edit: this is in regard to vpred/edm2 training

**Comment by Fear-in-a-hand-full-of-dust** [comment]
> Hey [@Fear-in-a-hand-full-of-dust](https://github.com/Fear-in-a-hand-full-of-dust) , could you share the full stack trace if you are getting an error, any manifestations of it being "broken" (outputs, training metrics, etc) and the config?

Nothing is "broken", no errors, I'm just suddenly getting bad training results with a config that worked well in the past. The config I'm using is [config23.txt](https://github.com/user-attachments/files/19572061/config23.txt) which is just slightly different from the one you shared here https://github.com/user-attachments/files/18470185/recent_config.txt

**Comment by Fear-in-a-hand-full-of-dust** [comment]
> [@Fear-in-a-hand-full-of-dust](https://github.com/Fear-in-a-hand-full-of-dust) , ah, pretty sure I know the issue, there was a bug with adaptive clipping where it would amplify gradients, not clip them, fixed that a month or two ago iirc. So what you should do is lower weight decay, so say, 3e-6 with stable weight decay, and adaptive clipping to ~0.18 to start.

That fixed it, thanks a lot. Still think that a folder with some example configs wouldn't hurt though.
